tests/mocks/remote-signer.ts

```diff
@@ -43,12 +43,31 @@ export function createSuccessRemoteServiceMock(host = MOCK_HOST) {
   );
 }
 
+export function createFailureRemoteServiceMock(host = MOCK_HOST) {
```

Comment:
Not using this in the tests, but thought it'd still be useful. Lemme know if I should remove it!

Reply:
It's fine to leave as is.